config/initializers/stripe.rb

```diff
@@ -4,3 +4,8 @@
 }
 
 Stripe.api_key = Rails.configuration.stripe[:secret_key]
+
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded', CreatePayment.new
```

Comment:
Very nice, I like how you created services to receive these calls. I was wondering where I would see those used.